app/services/preserver/importer.rb

```diff
@@ -11,23 +11,16 @@ def self.default_storage_adapter
     # @param resource [PreservationObject]
     def self.from_preservation_object(resource:, change_set_persister:, storage_adapter: nil)
       file_metadata = resource.metadata_node
-      metadata_file_identifier = if file_metadata.nil?
-                                   nil
-                                 else
-                                   file_metadata.file_identifiers.first
-                                 end
-      binary_nodes = resource.binary_nodes
-      binary_file_identifiers = binary_nodes.map(&:file_identifiers)
-      binary_file_identifiers.flatten!
-
+      metadata_file_identifier = file_metadata.try(:file_identifiers).try(:first)
```

Comment:
Is this `.try` because it can be nil? If so, can we do `file_metadata&.file_identifiers&.first`?

Reply:
Good catch. I'm not sure why this would every be an issue, but there is a test for it and this line is a compact version of the previous logic.